src/test/java/bio/terra/service/snapshot/SnapshotServiceTest.java

```diff
@@ -125,12 +137,16 @@ public class SnapshotServiceTest {
   @MockBean private AzureSynapsePdao synapsePdao;
   @MockBean private EcmService ecmService;
   @MockBean private RawlsService rawlsService;
+  @MockBean private DuosClient duosClient;
 
   private final UUID snapshotId = UUID.randomUUID();
   private final UUID datasetId = UUID.randomUUID();
   private final UUID snapshotTableId = UUID.randomUUID();
   private final UUID profileId = UUID.randomUUID();
+  private final UUID duosFirecloudGroupId = UUID.randomUUID();
   private final Instant createdDate = Instant.now();
+  private final DuosFirecloudGroupModel duosFirecloudGroup =
+      DuosFixtures.duosFirecloudGroupFromDb(DUOS_ID, duosFirecloudGroupId);
```

Comment:
Ahhh, I can see how this naming is confusing.  This fixture returns a `DuosFirecloudGroupModel` populated in a manner matching the guarantees of a record retrieved from the DB (ex. has id populated), but it doesn't make a DB call to get it.

Would `mockDuosFirecloudGroupFromDb` be clearer, or do you have another naming suggestion to disambiguate?  (I am not interacting with mocks in this call either, so wasn't sure if that presented a different point of confusion.)

Reply:
Good call, I've added javadocs to these methods. We have a mix of live resource creation and mocked object creation in fixtures, it would be nice to have clearer delineation between what you're getting.  Thanks for the naming thoughts, I've been staring at this for too long!